manager/integration/deploy/test.yaml

```diff
@@ -87,6 +87,8 @@ spec:
       valueFrom:
         fieldRef:
           fieldPath: spec.nodeName
+    - name: MANAGED_K8S_CLUSTER
+      value: "true"
```

Comment:
LGTM. But need to set `MANAGED_K8S_CLUSTER` to `false` to avoid it running by default.